common-lib/src/main/java/com/google/gct/idea/util/PlatformInfo.java

```diff
@@ -0,0 +1,32 @@
+package com.google.gct.idea.util;
+
+import com.intellij.openapi.application.ApplicationInfo;
+import com.intellij.util.PlatformUtils;
+
+import org.jetbrains.annotations.NotNull;
+
+public class PlatformInfo {
+
+  @NotNull
+  public static IntelliJPlatform getCurrentPlatform() {
+    return IntelliJPlatform.fromPrefix(PlatformUtils.getPlatformPrefix());
+  }
+
+  @NotNull
+  public static String getCurrentPlatformName() {
+    return getCurrentPlatform().getName();
+  }
+
+  @NotNull
+  public static String getUserAgent() {
+    // todo read this from build.gradle
+    String userAgent = "GCP-Plugin-for-IntelliJ/0.91";
+    try {
+      userAgent += " " + getCurrentPlatformName()
```

Comment:
shouldn't we cache this? I guess it doesn't change during the lifetime of the application.